client/my-sites/current-site/index.jsx

```diff
@@ -42,6 +42,38 @@ class CurrentSite extends Component {
 		this.props.recordTracksEvent( 'calypso_sidebar_all_sites_click' );
 	};
 
+	renderSiteSwitcher = () => {
+		const { translate, isRtl } = this.props;
+		const arrowDirection = isRtl ? 'right' : 'left';
+
+		if ( isEnabled( 'layout/dotcom-nav-redesign' ) ) {
+			return (
+				<span className="current-site__switch-sites">
```

Comment:
This code duplication is "intended"; I tried refactoring it but it is uglier than just copy-pasting the code 😅